tests/Application.UnitTests/Common/Exceptions/ValidationExceptionTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using CleanArchitecture.Application.Common.Exceptions;
+using FluentAssertions;
+using FluentValidation.Results;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace CleanArchitecture.Application.UnitTests.Common.Exceptions
+{
+    public class ValidationExceptionTests
+    {
+        [Test]
+        public void DefaultConstructorCreatesAnEmptyErrorDictionary()
+        {
+
+
+            var actual = new ValidationException().Errors;
+
+
```

Comment:
There are some emtpy lines that could be removed

Reply:
That is my own private unit test shorthand. Instead of inserting comments for Arrange, and Act, Assert I use two blank lines. Not appropriate for other peoples repositories. 

I will copy the the styling from other unit tests in this project.

Reply:
All fixed.